components/dashboard/src/projects/NewProject.tsx

```diff
@@ -193,33 +218,38 @@ export default function NewProject() {
 
     const renderSelectRepository = () => {
 
+        const noReposAvailable = reposInAccounts.length === 0;
+        const filteredRepos = Array.from(reposInAccounts).filter(r => r.account === selectedAccount && r.name.includes(repoSearchFilter));
         const icon = selectedAccount && accounts.get(selectedAccount)?.avatarUrl;
 
-        const renderRepos = () => (<div className="mt-10 border rounded-t-xl border-gray-100 flex-col">
-            <div className="px-8 pt-8 flex flex-col space-y-2">
-                <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
-                    <div className="w-full">
-                        <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
-                        <input className="w-full px-12 cursor-pointer font-semibold" readOnly type="text" value={selectedAccount || ""}></input>
-                        <img src={CaretDown} title="Select Account" className="filter-grayscale absolute top-1/2 right-3" />
+        const renderRepos = () => (<>
+            <div className={`mt-10 border ${isGitHub() ? "rounded-t-xl border-b-0" : "rounded-xl"} border-gray-100 flex-col`}>
+                <div className="px-8 pt-8 flex flex-col space-y-2">
+                    <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
+                        <div className="w-full">
+                            {icon && (
+                                <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
+                            )}
+                            <input className="w-full px-12 cursor-pointer font-semibold" readOnly type="text" value={selectedAccount || ""}></input>
```

Comment:
**issue**: Is it expected that group avatars fail to load for groups? It seems that shared projects with write access on personal accounts successfully fetch the avatars.

| Missing avatars on groups (a) | Missing avatars on personal accounts (b)|
|-|-|
| <img width="840" alt="Screenshot 2021-08-17 at 5 44 31 PM" src="https://user-images.githubusercontent.com/120486/129751165-38363e59-9b1a-4b19-9e12-0fe0aa05e33a.png"> | <img width="772" alt="Frame 34" src="https://user-images.githubusercontent.com/120486/129752105-ae42c776-e95f-4775-9efc-957f610f061e.png"> |

Reply:
indeed :-(

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/5330.